exporter/sentryexporter/sentry_exporter.go

```diff
@@ -361,21 +369,25 @@ func statusFromSpanStatus(spanStatus pdata.SpanStatus) (status sentry.SpanStatus
 	return canonicalCodes[code], spanStatus.Message()
 }
 
-// isRootSpan determines if a span is a root span.
-// If parent span id is empty, then the span is a root span.
-func isRootSpan(s *sentry.Span) bool {
-	return s.ParentSpanID == sentry.SpanID{}
+// spanIsTransaction determines if a span should be sent to Sentry as a transaction.
+// If parent span id is empty or the span kind allows remote parent spans, then the span is a root span.
+func spanIsTransaction(s pdata.Span) bool {
+	kind := s.Kind()
+	return s.ParentSpanID() == pdata.SpanID{} || kind == pdata.SpanKindServer || kind == pdata.SpanKindConsumer
```

Comment:
Can we make sure to update https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/sentryexporter/docs/transformation.md this new context?

Also, can we link to the opentelemetry spec about span kind in the code comments? Like what we do here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/21b423bfff56ca1e5d50e6b8f1f9f4929603695b/exporter/sentryexporter/sentry_exporter.go#L273